scss/iframes/_index.scss

```diff
@@ -1,3 +1,4 @@
 @forward 'base';
 @forward 'font';
 @forward 'size';
+@forward 'color';
```

Comment:
Since all the code in this file was only being used in iframes, I moved this file here with the rest of the iframe preview files.